src/content/en/fundamentals/media/mse/basics.md

```diff
@@ -91,6 +91,11 @@ Here, in no particular order, are a few things I won't cover.
    `<audio>` and `<video>` elements.
 +  Error handling for the sake of simplicity and clarity.
 
+Warning: Handling error events, API exceptions, and proactively checking
+`HTMLMediaElement readyState` and `MediaSource readyState` values before making
+calls on their APIs are mandatory in a production usage of the APIs as these
+values can change before associated events are delivered.
+
```

Comment:
This is a bit of a mouthful. Can you please take another shot at it. Here's a suggestion:

"Before making calls on these APIs, handle any error events or API exceptions, and check `HTMLMediaElement readyState` and `MediaSource readyState`. These values can change before associated events are delivered.